back/node_watcher/kubernetes/staging/deployments/nomidotwatcher-deployment.yaml

```diff
@@ -27,4 +27,7 @@ spec:
           env:
             - name: PRISMA_ENDPOINT
               value: http://10.0.9.43:4466
-          command: ["yarn", "start"]
+          lifecycle:
+            postStart:
+              exec:
+                command: ["yarn", "start"]
```

Comment:
I'm not 100% sure what this means, but I'll trust you!

Reply:
ah just tested it out but this is actually not necessary